Client/WebAuthN/U2FExtensions.swift

```diff
@@ -88,6 +94,8 @@ class U2FExtensions: NSObject {
     
     fileprivate static var observationContext = 0
     
+    fileprivate var nfcSesionStateObservation: NSKeyValueObservation?
```

Comment:
Should use the most strict accessor for the required scope.
For me, it looks like these should be `private` instead of `fileprivate`